common/list/_lists.scss

```diff
@@ -77,3 +75,56 @@ ul.list-col-xs-1 > li { flex-basis: 100% }
 ul.list-col-xs-2 > li { flex-basis: 50% }
 ul.list-col-xs-3 > li { flex-basis: 33.33% }
 ul.list-col-xs-4 > li { flex-basis: 25% }
+
+
+div.lst-lwr-alph>ol,
+.lst-lwr-alph {
+	list-style-type: lower-alpha;
+}
+
+div.lst-upr-alph>ol,
+.lst-upr-alph {
+	list-style-type: upper-alpha;
+}
+
+div.lst-lwr-rmn>ol,
+.lst-lwr-rmn {
+	list-style-type: lower-roman;
+}
+
+div.lst-upr-rmn>ol,
+.lst-upr-rmn {
+	list-style-type: upper-roman;
+}
+
+div.lst-num>ol,
```

Comment:
I understand that we did had that once already in our code base, so it makes sense to keep it here. My concern is do we need to adapt it for the edge case too? What is the use case or an working example that illustrate that need.

Reply:
Removed. Thanks